src/main/frontend/components/content.cljs

```diff
@@ -168,11 +158,12 @@
         [:.menu-links-wrapper
          [:div.flex-row.flex.justify-between.pb-2.pt-1.px-2
           [:div.flex-row.flex.justify-between
-           (for [color block-background-colors]
+           (for [color ui/block-background-colors]
              [:a.m-2.shadow-sm
-              {:on-click (fn [_e]
+              {:title (t (keyword "color" color))
+               :on-click (fn [_e]
                            (editor-handler/set-block-property! block-id "background-color" color))}
-              [:div.heading-bg {:style {:background-color color}}]])
+              [:div.heading-bg {:style {:background-color (str "var(--color-" color "-500)")}}]])
```

Comment:
We are using a naming convention based on the `block-background-colors` vector.
A vector of maps that explicitly sets the name of the color and the css properties could be more robust/open, but it feels a little verbose.
```
(def block-background-colors
   [{:id "gray"
     :title (t :color/gray)
     :default-color "var(--color-gray-500)" 
     :highlight-color "var(--ls-highlight-color-gray)"}
   ...])
```

Also, we are currently storing the color as a string (eg `red`) We could use a key instead of a string to differentiate between plain color values and keys that will be translated to actual colors by our system. The downside is that this is not backward compatible, and third party apps won't know how to handle keys, which shouldn't be a big deal.

@logseq-cldwalker It would be great to hear your opinion on both issues. It is going to be even harder to change this system again, when it goes out.

Reply:
> The downside is that this is not backward compatible, and third party apps won't know how to handle keys, which shouldn't be a big deal.

I don't know what third-party apps depend on for color values and if this breaking change would be a big deal or not. Seems like a good question for @xyhp915